### PR TITLE
[browser] Allow lipstick to activate new tab view. Fixes JB#54319 OMP#JOLLA-152

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by rainemak** [comment]
CC @dseight as well.

**Review by dseight — APPROVED** [review]

**Comment by rainemak** [comment]
Still ok?

File: apps/browser/browserservice.cpp [hunk 1]
```diff
@@ -26,6 +26,8 @@ const auto ProcDir = QStringLiteral("/proc/%1");
 const auto ErrorPidIsNotPrivileged = QStringLiteral("PID %1 is not in privileged group");
 const auto SailfishBrowserUiService = QStringLiteral("org.sailfishos.browser.ui");
 const auto TransferEngine = QStringLiteral("org.nemo.transferengine");
+const auto Lipstick = QStringLiteral("com.jolla.lipstick");
+const auto DBusErrorMessage = QStringLiteral("PID %1 is not the owner of '%2' or in privileged group");
```
Comment (Tomin1) [inline apps/browser/browserservice.cpp:30]: Looks good but you could have a bit more descriptive name here. Note that there is another error message a few lines up.

**Review by Tomin1 — APPROVED** [review]
LGTM still. One comment about variable naming still though. I think I know how to reproduce the bug so I'll file a report for it later.

File: apps/browser/browserservice.cpp (hunk 1 above)
Comment (rainemak) [reply apps/browser/browserservice.cpp:30]: Will update to ErrorPidIsNotOwnerServiceOrPrivileged

**Pull request merged**